test/xbgpu/test_engine.py

```diff
@@ -1011,6 +1010,7 @@ def engine_arglist(
         corrprod_args: list[str],
         beam_args: list[str],
     ) -> list[str]:
+        """Command-line arguments for the engine."""
```

Comment:
Could probably use the same thing for `fgpu/test_engine.py`'s engine-arglist fixture.